src/todolist/settings.py

```
@@ -18,7 +18,7 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
+SECRET_KEY = os.getenv("SECRET_KEY", "")
```

Current code uses `os.getenv("SECRET_KEY", "")`, which will silently set SECRET_KEY to an empty string if the environment variable is missing. The task requires the application to use the Secret instead of a hardcoded value and to fail-fast or otherwise not run with an empty secret. Change this to a fail-fast pattern such as `SECRET_KEY = os.environ['SECRET_KEY']` or explicitly raise an error if the env var is missing so misconfiguration is visible to reviewers and automation. See this file for the current line.